desc/compute/_geometry.py

```diff
@@ -196,7 +196,7 @@ def _A_of_z_FourierRZToroidalSurface(params, transforms, profiles, data, **kwarg
             transforms["grid"],
             data["Z"] * n[:, 2] * jnp.sqrt(data["g_tt"]),
             line_label="theta",
-            fix_surface=("rho", 1.0),
```

Comment:
do we also want to scale the area up if the largest rho isnt 1?

Reply:
I don't think that makes sense. It's proportional sure but the integrand will be different, so it's not exact.

Reply:
Was our justification to do that for `perimeter(z)` because we assume integrand doesn't change much near edge of quadrature node and the lcfs?

Reply:
Yeah pretty much. Obviously if you're using the rho=0.5 surface it's a bad guess, but for rho near 1 it should be reasonably accurate (and exact for elliptical cross section)